exporter/otlpexporter/README.md

```diff
@@ -15,7 +15,9 @@ The following settings are required:
 
 - `endpoint` (no default): host:port to which the exporter is going to send OTLP trace data,
 using the gRPC protocol. The valid syntax is described
-[here](https://github.com/grpc/grpc/blob/master/doc/naming.md)
+[here](https://github.com/grpc/grpc/blob/master/doc/naming.md).
+Including a scheme of `http` or `https` is allowed in the `endpoint`, but not required.
+If a scheme of `https` is used then client transport security overriding the `insecure` setting.
```

Comment:
Can you please clarify this sentence? I am not sure I understand it.

Reply:
Oops, yes I've left out some words, so this sentence makes no sense! Fixed.

The idea is that if you use the scheme `https` then the `insecure` setting is ignored. This is inline with the outstanding spec PR https://github.com/open-telemetry/opentelemetry-specification/pull/1729